docs/changelog.md

```diff
@@ -2,6 +2,14 @@
 
 ## CARET
 
+### v0.3.1 <small>\_ Oct 31, 2022</small> {id = "0.3.1"}
+
+- Added record function to ros2caret for upcoming feature. ([ros2caret #38](https://github.com/tier4/ros2caret/pull/38))
+- Added function to display histogram for response time ([CARET_analyze #165](https://github.com/tier4/CARET_analyze/pull/165))
+- Added API to get period and response time plotted by time-series ([CARET_analyze #167](https://github.com/tier4/CARET_analyze/pull/167), [CARET_analyze #168](https://github.com/tier4/CARET_analyze/pull/168))
```

Comment:
@hsgwa 
I think APIs now described in PR 167 and 168 are not for response time.
These PRs are adapting the latency of message flow, aren't they?

Reply:
> These PRs are adapting the latency of message flow, aren't they?

https://github.com/tier4/CARET_analyze/pull/167 and  https://github.com/tier4/CARET_analyze/pull/168 are not only for message flow.
The PRs add calcuration api for latency and period which is used for callback, communication, and so on.

They are refactor and quality related PRs rather than new feature.

Reply:
There are some candidates to express `get latency and period`;

- calculate,
- get,
- retrieve, etc.

Reply:
I agree "calculate" and "get".

Reply:
I suggested diff in: https://github.com/tier4/CARET_doc/pull/103#discussion_r1015056541

Reply:
I received a comment in [here](https://github.com/tier4/CARET_doc/pull/103#issuecomment-1305436709)

But it looks like no change.
`Added API to get period and response time plotted by time-series` <br>
↓<br>
`Added API to calculate period and latency plotted by time-series`

Reply:
@takam5f2